### PR TITLE
Adding WAITING status in checkstatusLoop

### PR DESCRIPTION
**Pull request by aspiringmind-code**

### CODE REVIEW

File: src/python/CRABClient/ClientUtilities.py [hunk 1]
```diff
@@ -760,10 +760,13 @@ def checkStatusLoop(logger, server, api, taskname, targetstatus, cmdname):
         elif taskStatus in ['NEW', 'HOLDING', 'QUEUED', 'RESUBMIT']:
             logger.info("Please wait...")
             time.sleep(30)
+        elif taskStatus in ['WAITING']:
```
Comment (belforte) [inline src/python/CRABClient/ClientUtilities.py:763]: I think that this message does not add value here, better simply to add WAITING to the list at line 760
A different story is to write something useful from `crab status`. Also we will remove all of this as we refactor the way to do dry run as per https://github.com/dmwm/CRABServer/issues/6544

**Review by belforte — CHANGES_REQUESTED** [review]

**Comment by belforte** [comment]
you can ignore pylint in CRABClient. Most  of the complain are due to the fact that we need to keep code compatible with python2

**Review by belforte — APPROVED** [review]

**Comment by belforte** [comment]
perfect, thanks.

File: src/python/CRABClient/ClientUtilities.py (hunk 1 above)
Comment (aspiringmind-code) [reply src/python/CRABClient/ClientUtilities.py]: Ok, separate elif clause for WAITING removed

**Pull request merged**